<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOencvWidget</class>
 <widget class="QWidget" name="QOencvWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,100">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <spacer name="horizontalSpacer_25">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="current_label_x">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="current_linedit_x">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="current_label_y">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="current_linedit_y">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="current_label_rgb">
           <property name="text">
            <string>RGB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="current_linedit_rgb">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>8</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="img_widget" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
